src/Linters/NoWhitespaceAtEndOfLineLinter.php

```diff
@@ -21,12 +21,19 @@
   public function getLintErrors(): Traversable<FixableLineLintError> {
     $lines = $this->getLinesFromFile();
     $errs = vec[];
+
     foreach ($lines as $ln => $line) {
       for ($i = strlen($line) - 1; $i >= 0; $i--) {
         $char = $line[$i];
         if ($char !== ' ') {
           break;
         }
+
+        // Looks like we have an error. Let's see if we should ignore this one
+        if ($ln-1 >= 0 && Str\contains($lines[$ln-1], $this->markerFixMe())) {
```

Comment:
AutofixingLineLinter ?  though that might need a refactor that'd be better done separately to make it so that the base does the iteration, and subclasses only look at one line at a time. cc @bradleyhd

Reply:
yeah - should probably even be up one more in LineLinter. Alternatively, what do you think about making whether or not the error should be ignored a (derived?) property of the `LintError` itself, and then just `Vec\filter`ing out as necessary in `LinterCLI` ex post facto?

Reply:
maybe base classes in BaseLinter, but if you’re integrating these into something else without the CLI, you shouldn’t have to explicitly check for ignored ones.

Reply:
Fair!